packages/components/src/components/carousel/carousel.scss

```diff
@@ -22,7 +22,7 @@
         height: 100%;
     }
     &__card {
-        display: flex;
+        display: grid;
```

Comment:
@ali-fs , why do we need to switch this to css grids?

Reply:
@ali-fs Yes, it does not make sense to me too. can you explain the reason behind this change?